apps/labs/posts/pytorch-tensoriterator-internals.md

```diff
@@ -0,0 +1,274 @@
+---
+title: 'PyTorch TensorIterator Internals'
+published: April 13, 2020
+author: sameer-deshmukh
+description: 'This post is a deep dive into how TensorIterator works, and is an essential part of learning to contribute to the PyTorch codebase since iterations over tensors in the C++ codebase are extremely commonplace. This post is aimed at someone who wants to contribute to PyTorch'
+category: [Machine Learning]
+featuredImage:
+  src: posts/pytorch-tensoriterator-internals/feature.png
+  alt: 'Code snippet on creating a TensorIterator using a default constructor.'
+hero:
+  imageSrc: posts/pytorch-tensoriterator-internals/blog_hero_var2.svg
```

Comment:
```suggestion
  imageSrc: posts/pytorch-tensoriterator-internals/blog_hero_var1.svg
```

Change file path to fix blank image, based on [recent blog post build](https://quansight-labs-live-git-pytorch-tensoriterator-3c59ee-quansight.vercel.app/blog/pytorch-tensoriterator-internals).